changelog.d/5734.bugfix

```diff
@@ -0,0 +1,3 @@
+The decorated servlets were producing cryptic names in prometheus after the
```

Comment:
sorry, this is probably too wordy now :). Also the release number is probably more relevant to most readers than the

How about:

> Fix a regression introduced in v1.2.0rc1 which led to incorrect labels on some prometheus metrics.

Reply:
Perfect